packages/core/src/shared/clients/codecatalystClient.ts

```diff
@@ -191,7 +203,7 @@ function fixAliasInRequest<
     return request
 }
 
-class CodeCatalystClientInternal {
+export class CodeCatalystClientInternal {
```

Comment:
does this need to be exported? I don't see a use in the rest of the PR
```suggestion
class CodeCatalystClientInternal {
```

Reply:
Leftover from debugging. Good catch